.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -7,6 +7,7 @@ body:
       value: |
         Hello Superset Community member! Please keep things tidy by putting your post in the proper place:
 
+        🚨 Reporting a security issue: send an email to security@apache.org. DO NOT USE GITHUB ISSUES TO REPORT SECURITY PROBLEMS.
```

Comment:
Is there a way to make that last sentence in a 256pt blinking red font?

Reply:
let's change the email to security@superset.apache.org, inline with: https://github.com/apache/superset/security/policy